### PR TITLE
Contact detail component

### PR DESCRIPTION
**Pull request by MariaRiosNavarro**
Please check:

- StyledLinkSVGContainer created  and to use it  in ContactListItem (Svg assets in new assets folder: contactsSVG)
- Route /contacts/[dynamicId] created . Inside the ContactDetail component to render dynamically (useRouter)
- Add the logic to make dynamic performance possible in index.js too
 
[Deployment](https://mobile-simulation-prop-11w0f9qg5-mariariosnavarro.vercel.app/)

### CODE REVIEW

File: pages/contacts/[dynamicId]/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+const StyledInfo = styled.p`
+  background-color: lightgray;
+  padding: 1rem;
+  border-radius: 8px;
+`;
+
+export default function ContactDetail() {
+  const router = useRouter();
+  const { dynamicId } = router.query;
+  const contactIndex = contacts.findIndex(
```
Comment (actyvystom) [inline pages/contacts/[dynamicId]/index.js]: You could easily use the `.find()` array method here, so can get rid of your intermediate variable `contactIndex`: 
```js
const contact = contacts.find((contact)=>contact.id === dynamicId);
```

**Review by actyvystom — CHANGES_REQUESTED** [review]
All in all, nice work! 👍🏻 
Just a tiny issue to compact some code.
Cheers, Thomas

File: pages/contacts/[dynamicId]/index.js (hunk 1 above)
Comment (MariaRiosNavarro) [reply pages/contacts/[dynamicId]/index.js]: done

**Review by ghost — APPROVED** [review]

**Comment by MariaRiosNavarro** [comment]
I almost closed without having done the merge ...
that's why I opened the PR again. I do the merge before. 😅

**Pull request merged**